Juice.xcodeproj/project.pbxproj

```diff
@@ -598,7 +630,7 @@
 				CODE_SIGN_IDENTITY = "Mac Developer";
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 529;
-				DEVELOPMENT_TEAM = YN24FFRTC8;
+				DEVELOPMENT_TEAM = A925WRGBAZ;
```

Comment:
I can't accept these changes to the development team, sorry it need so to continue to be the old development team.